python/Ganga/Lib/Localhost/LocalHostExec.py.template

```diff
@@ -76,6 +76,8 @@ sys.path.insert(0, ###GANGADIR###)
 sys.path.insert(0,os.path.join(os.getcwd(),PYTHON_DIR))
 
 fullenvironment = os.environ.copy()
+for key,value in environment.iteritems():
+    fullenvironment[key] = value
 
```

Comment:
`environment` contains the env variables required by the job (i.e. set in the RT Handler of the application). What this does is combine the current environment with these additional variables. Without it, they aren't set and the Local backend will fail for applications that need additional env variables set (that may just be Atlas ones, but best to be consistent!).